federation/executor.go

```diff
@@ -159,10 +274,10 @@ func (e *Executor) execute(ctx context.Context, p *Plan, keys []interface{}) (in
 			resMu.Lock()
 			defer resMu.Unlock()
 			for k, v := range result {
-				if _, ok := subPlanMetaData.results[k]; ok {
-					return oops.Errorf("key already exists in results: %v", k)
+				if _, ok := subPlanMetaData.results[k]; !ok {
+					subPlanMetaData.results[k] = v
```

Comment:
Is it easier to delete the __federation key here?

Reply:
Yeah, sure!